spec/rspec/rails/matchers/action_cable/have_broadcasted_to_spec.rb

```diff
@@ -226,5 +226,43 @@ def broadcast(stream, msg)
         end
       end
     end
+
+    it "has a description when not chained" do
+      expect(have_broadcasted_to("stream").description).to eq("have broadcasted exactly 1 messages to stream")
```

Comment:
“stream” reads too generic, maybe “my_stream” to make it clear that it’s the same as specified?